### PR TITLE
Zerberus loop

### PR DESCRIPTION
**Pull request by hpfmn**
this adds loop support - fixes #114681

once everything is good I'll squash commits together!

### CODE REVIEW

File: audiofile/audiofile.cpp [hunk 1]
```diff
@@ -61,6 +62,11 @@ bool AudioFile::open(const QByteArray& b)
       buf = b;
       idx = 0;
       sf  = sf_open_virtual(&sfio, SFM_READ, &info, this);
+      if (sf_command(sf, SFC_GET_INSTRUMENT, &inst, sizeof(inst)) == SF_TRUE)
+            hasInstrument = true;
+      else
+            hasInstrument = false;
+
```
Comment (Jojo-Schmitz) [inline audiofile/audiofile.cpp]: how about
`hasInstrument = sf_command(sf, SFC_GET_INSTRUMENT, &inst, sizeof(inst)) == SF_TRUE`

Comment (hpfmn) [inline audiofile/audiofile.cpp]: good idea

File: zerberus/sfz.cpp [hunk 2]
```diff
@@ -188,8 +193,15 @@ void ZInstrument::addRegion(SfzRegion& r)
                   }
             }
       Zone* z = new Zone;
-      r.setZone(z);
       z->sample = readSample(r.sample, 0);
+      if (z->sample) {
+            qDebug("Sample Loop - start %d, end %d, mode %d", z->sample->loopStart(), z->sample->loopEnd(), z->sample->loopMode());
+            if (r.loopStart == -1)
```
Comment (lasconic) [inline zerberus/sfz.cpp]: Could you add a comment here? Something like read "loop from sample if no loop in defined in the region". Is this in the spec btw?

**Comment by lasconic** [comment]
If I understand correctly this PR https://github.com/musescore/MuseScore/pull/2670 supersede this one.
Correct? If yes, I will close it.

**Comment by hpfmn** [comment]
Besides the the comment it contains the same changes

**Pull request closed without merging**